contxt/cli/commands/ems.py

```diff
@@ -35,36 +35,46 @@ def mains(
     clients: Clients, facility_id: int, resource_type: ResourceType, fields: List[str], sort: str
 ) -> None:
     """Get main services"""
-    items = clients.ems.get_main_services(facility_id=facility_id, resource_type=resource_type)
+    results = clients.nionic.get_main_services(facility_id=facility_id, resource_type=resource_type)
+    items = [
+        MainService(
+            x.id, x.facility_id, x.name, x.type, x.demand.id, x.usage.id, x.created_at, x.updated_at
+        )
+        for x in results
+    ]
     print_table(items=items, keys=fields, sort_by=sort)
 
 
 @ems.command()
 @click.argument("facility_id")
 @click.option("--resource-type", type=ResourceType, default="electric", help="Resource type")
-@click.option("--start", type=click.DateTime(), help="Start time")
-@click.option("--end", type=click.DateTime(), help="End time")
+@click.option("--start", type=click.DateTime(), help="Start time", required=True)
+@click.option("--end", type=click.DateTime(), help="End time", required=True)
 @click.pass_obj
 def main_data(
     clients: Clients, facility_id: int, resource_type: ResourceType, start: datetime, end: datetime
 ) -> None:
     """Get main service data"""
     data: Dict[datetime, Dict[str, Any]] = defaultdict(dict)
-    services = clients.ems.get_main_services(facility_id=facility_id, resource_type=resource_type)
+    services = clients.nionic.get_main_services(facility_id=facility_id, resource_type=resource_type)
     with click.progressbar(
         services,
         label="Downloading main service data",
         item_show_func=lambda s: f"Service {s.name}" if s else "",
     ) as services_:
         for service in services_:
-            for t, v in clients.iot.get_time_series_for_field(
-                field=service.usage_field,
-                start_time=start,
-                end_time=end,
+            for t, v in clients.nionic.get_data_point_data(
+                data_source_name=service.usage.data_source_name,
+                name=service.usage.name,
+                start=start.astimezone().isoformat(),
+                end=end.astimezone().isoformat(),
                 window=Window.MINUTELY,
                 per_page=5000,
             ):
-                data[t][service.usage_field.field_human_name] = v
+                if t in data and service.usage.alias in data[t]:
+                    data[t][service.usage.alias] = v + data[t][service.usage.alias]
```

Comment:
before, if there were multiple mains for a given facility, with some of them having the same `field_human_name`, the usage was not getting summed up. That seems like a bug, but I want to call it out in case my assumption is wrong